packages/grid/theme/lumo/vaadin-grid-styles.js

```diff
@@ -69,12 +69,14 @@ registerStyles(
     }
 
     [part~='row']:focus,
-    [part~='cell']:focus {
+    [part~='cell']:focus,
+    [part~='cell']:not([tabindex]) ::slotted(vaadin-grid-cell-content:focus) {
       outline: none;
     }
 
     :host([navigating]) [part~='row']:focus::before,
-    :host([navigating]) [part~='cell']:focus::before {
+    :host([navigating]) [part~='cell']:focus::before,
+    :host([navigating]) [part~='cell']:not([tabindex]):focus-within::before {
```

Comment:
Looks like this introduced a glitch. For editable cells it now shows the native browser outline, instead of the Lumo outline:

Chrome | Safari | Firefox
-- | -- | --
 <img width="358" alt="Bildschirmfoto 2022-07-04 um 11 49 54" src="https://user-images.githubusercontent.com/357820/177130940-231c734e-311f-4b87-bc91-fcfb9f4eac73.png"> | ![Bildschirmfoto 2022-07-04 um 11 50 01](https://user-images.githubusercontent.com/357820/177131007-58c0ee3f-0776-44d8-a5f8-1ad74831caa5.png)  |  ![Bildschirmfoto 2022-07-04 um 11 50 24](https://user-images.githubusercontent.com/357820/177130874-e8b5d454-1cf9-4e2e-97be-e550ea911345.png)

The Lumo outline is only visible when selecting a cell by mouse click, and the mouse button is pressed down (active state). Also just clicking the cell does not show the native outline, it only becomes visible after moving the focus by keyboard. On `master` the outline is immediately visible when clicking the cell, which I guess is different for editable cells in Grid Pro, when compared to regular Grid.